metrics.go

```diff
@@ -14,9 +13,8 @@ import (
 func (c *collector) clusterMetrics(ch chan<- prometheus.Metric, client *kazoo.Kazoo) {
 	controller, err := client.Controller()
 	if err != nil {
-		msg := fmt.Sprintf("Error collecting cluster controller broker ID: %s", err)
-		log.Error(msg)
-		ch <- prometheus.NewInvalidMetric(prometheus.NewDesc("zookeeper_controller_id_error", msg, nil, nil), err)
+		log.Errorf("Error collecting cluster controller broker ID: %s", err)
```

Comment:
I still want that metric if we can get it and if it's useful, but later is totally valid.